ci/axis/miniforge-cuda-driver-arm64.yml

```diff
@@ -33,28 +31,6 @@ DRIVER_VER:
   - 495
 
 exclude:
-  - CUDA_VER: 11.0
-    DRIVER_VER: 455
-  - CUDA_VER: 11.0
-    DRIVER_VER: 460
-  - CUDA_VER: 11.0
-    DRIVER_VER: 465
-  - CUDA_VER: 11.0
-    DRIVER_VER: 470
-  - CUDA_VER: 11.0
-    DRIVER_VER: 495
-
-  - CUDA_VER: 11.1
-    DRIVER_VER: 450
-  - CUDA_VER: 11.1
-    DRIVER_VER: 460
-  - CUDA_VER: 11.1
-    DRIVER_VER: 465
-  - CUDA_VER: 11.1
-    DRIVER_VER: 470
-  - CUDA_VER: 11.1
-    DRIVER_VER: 495
-
   - CUDA_VER: 11.2
     DRIVER_VER: 450
```

Comment:
As mentioned on [this page](https://docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html), driver versions `450` and `455` correspond to CUDA versions `11.0` and `11.1` respectively. Therefore, if we're not building `11.0` and `11.1` images for ARM, the respective driver versions should be removed throughout the ARM files as well.